src/functions/stopTimeout.js

```diff
@@ -3,16 +3,20 @@ module.exports = async (d) => {
     if (data.err) return d.error(data.err);
 
     const [id] = data.inside.splits;
+    const MAX_SAFE_INTEGER = 0x7FFFFFFF;
```

Comment:
@USERSATOSHI ?

Reply:
thats the max safe integer which prevents the timeout from overflowing

Reply:
> thats the max safe integer which prevents the timeout from overflowing

better use Number.MAX_SAFE_INTEGER ig?

Reply:
nvm that wont work , but better have different name, something like MAX_TIMEOUT_DURATION or something

Reply:
yop